tests/cases/compiler/es6modulekindWithES5Target.ts

```diff
@@ -0,0 +1,20 @@
+// @target: es5
+// @module: es2015
+// @experimentalDecorators: true
+
+export class C {
+    static s = 0;
+    p = 1;
+    method() { }
+}
+
+declare function foo(...args: any[]): any;
+@foo
+export class D {
+    static s = 0;
+    p = 1;
+    method() { }
+}
+
+class E { }
+export {E};
```

Comment:
Can you add usage of C and D to  verify exports.C and exports.D is not being used